packages/kit/src/runtime/server/page/render.js

```diff
@@ -148,8 +148,8 @@ export async function render_response({
 		}
 		// prettier-ignore
 		head += Array.from(css)
-				.map((dep) => `\n\t<link${styles.has(dep) ? ' disabled' : ''} rel="stylesheet" href="${options.prefix + dep}">`)
-				.join('');
+            .map((dep) => `\n\t<link${styles.has(dep) ? ' media="print"' : ''} rel="stylesheet" href="${options.prefix + dep}">`)
+            .join('');
```

Comment:
I did some testing in different browsers. Turns out `media="print"` or `media="(max-width: 0)"` doesn't disable downloading, as you noted. `disabled` _does_ disable downloading in Chrome and Safari, but not Firefox. But combining them — `disable media="..."` — disables downloading in all three.

On reflection I think `(max-width: 0)` is preferable to `print`, since it kind of says 'hello! i am a deliberate hack' to anyone who happens to look at the HTML

```suggestion
			.map((dep) => `\n\t<link${styles.has(dep) ? ' disabled media="(max-width: 0)"' : ''} rel="stylesheet" href="${options.prefix + dep}">`)
			.join('');
```